### PR TITLE
updated the manual to match new ADVI interface

### PR DESCRIPTION
**Pull request by akucukelbir**
#### Summary

Updates the cmdStan manual to match the new ADVI interface. 

See issue https://github.com/stan-dev/cmdstan/issues/230

#### Intended Effect

Bring the manual up to date.

#### How to Verify

Read.

#### Side Effects

None.

#### Reviewer Suggestion

@dustinvtran @syclik

### CODE REVIEW

File: src/docs/cmdstan-guide/getting-started.tex [hunk 1]
```diff
@@ -688,14 +695,16 @@ \subsection{Variational Inference}
 it indicates the default mean-field setting of the variational
 inference algorithm. It also indicates the default parameter sizes and
 tolerances for monitoring the algorithm's convergence. The second part
-of the output describes the progression of the algorithm. The evidence
+of the output describes the progression of the algorithm. An adaptation
+phases finda good value for the step size scaling parameter $\eta$. The
```
Comment (bob-carpenter) [inline src/docs/cmdstan-guide/getting-started.tex]: finda

Comment (akucukelbir) [inline src/docs/cmdstan-guide/getting-started.tex]: that's how i roll :) thanks bob, i'll fix it

File: src/docs/cmdstan-guide/running.tex [hunk 2]
```diff
@@ -1014,14 +1014,19 @@ \subsection{Variational Inference-Specific Commands}
...
 %
     \hiercmdarg{\indentarrow\indentarrow}{algorithm}{$$list element$$}
       {Variational inference algorithm}
       {Valid values: \ \code{meanfield, fullrank}}
       {Defaults to \code{meanfield}}
 %
+    \hiercmdarg{\indentarrow\indentarrow}{iter}{$$int$$}
+      {Maximum number of iterations}
+      {Valid values: \ $0 < \mbox{\code{iter}}$}
+      {Defaults to \code{10000}}
```
Comment (bob-carpenter) [inline src/docs/cmdstan-guide/running.tex]: That seems like a lot of default iterations for an optimization procedure.  But anyway, we can change these in the future if it turns out to look like it hangs to users.

**Comment by bob-carpenter** [comment]
Looks great, Alp --- thanks for updating the example.

**Comment by bob-carpenter** [comment]
There's nothing else in this pull request, so it's either a Travis snafu or an indication of a problem with something not in this pull request.

@syclik : would you mind taking a look and merging it if Travis is OK?

**Comment by syclik** [comment]
@bob-carpenter, yeah, I'm on it. I just restarted the build on Travis.

**Comment by syclik** [comment]
I'll wait for "finda" before merging.

File: src/docs/cmdstan-guide/running.tex [hunk 3]
```diff
@@ -1034,14 +1039,24 @@ \subsection{Variational Inference-Specific Commands}
 %
     \hiercmdarg{\indentarrow\indentarrow}{eta}{$$double$$}
       {Stepsize weighting parameter for adaptive stepsize sequence}
-      {Valid values: \ $0 < \mbox{\code{eta}} \leq 1.0$}
+      {Valid values: \ $0 < \mbox{\code{eta}}$}
       {Defaults to \code{0.1}}
```
Comment (bgoodri) [inline src/docs/cmdstan-guide/running.tex]: It says above that the default value of eta is 1 rather than 0.1

Comment (akucukelbir) [inline src/docs/cmdstan-guide/running.tex]: great catch! thanks ben!

**Comment by akucukelbir** [comment]
thanks @syclik !

**Pull request merged**